types-core/index.d.ts

```diff
@@ -395,24 +401,33 @@ declare module '@azure/functions-core' {
         collectionDouble?: RpcCollectionDouble | null;
 
         collectionSint64?: RpcCollectionSInt64 | null;
+
+        modelBindingData?: ModelBindingData | null;
     }
 
-    interface RpcCollectionSInt64 {
-        sint64?: (number | Long)[] | null;
+    interface RpcCollectionBytes {
+        bytes?: Uint8Array[] | null;
     }
 
     interface RpcCollectionString {
         string?: string[] | null;
     }
 
-    interface RpcCollectionBytes {
-        bytes?: Uint8Array[] | null;
-    }
-
     interface RpcCollectionDouble {
         double?: number[] | null;
     }
 
+    interface RpcCollectionSInt64 {
+        sint64?: (number | Long)[] | null;
+    }
+
+    interface ModelBindingData {
```

Comment:
Make sure you sync these changes to the worker repo's core types, which is the source of truth: https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/types-core/index.d.ts

Also, not sure if you were aware, but most of the "Rpc" types are auto-generated in the worker repo based on the protobuf api contract. When you run `npm run build` in the worker repo the rpc types are output to `dist/azure-functions-language-worker-protobuf/src/rpc.d.ts`. For the core api, we copy the minimal types we need and clean it up a tiny bit (remove unnecessary comments, usually rename it from `IModelBindingData` to `RpcModelBindingData`, etc.)